src/cap32.cpp

```diff
@@ -3347,7 +3348,7 @@ int cap32_main (int argc, char **argv)
       exit(-1);
    }
 
-   loadConfiguration(CPC, getConfigurationFilename()); // retrieve the emulator configuration
+   loadConfiguration(CPC, args.cfgFilePath != "" ? args.cfgFilePath : getConfigurationFilename()); // retrieve the emulator configuration
```

Comment:
I'd prefer args.cfgFilePath.empty() ? ...
Also, how does it behave if the file doesn't exist (If I remember well, it now works but I'm not 100% sure) ?